rotkehlchen/data/all_assets.json

```diff
@@ -12754,6 +12763,15 @@
         "symbol": "WINGS",
         "type": "ethereum token"
     },
+    "WIS": {
+        "coingecko": "experty-wisdom-token",
+        "ethereum_address": "0xDecade1c6Bf2cD9fb89aFad73e4a519C867adcF5",
+        "ethereum_token_decimals": 18,
+        "name": "Experty Wisdom Token",
+        "started": 1605558693,
```

Comment:
This also seems a little bit off: 1604695924 is the deployment timestamp of the ethereum contract: https://etherscan.io/tx/0x1fb141389e9c4e2529b99cb4c04d22283253ad9b38773c498f20c756906282f7

Reply:
Got it.  Thanks!